File: C/1010 - Cálculo Simples.c
/* 
   ==UserScript==
 @name         Cálculo Simples
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1010 - Cálculo Simples
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

#include <stdio.h>
main()
{
    int Codigo1, Quantidade1, Codigo2, Quantidade2;
    double Valor_Unitario1, Valor_Unitario2, TOTAL;
    scanf("%d%d%lf", &Codigo1, &Quantidade1, &Valor_Unitario1);
    scanf("%d%d%lf", &Codigo2, &Quantidade2, &Valor_Unitario2);
    TOTAL = (Valor_Unitario1 * Quantidade1) + (Valor_Unitario2 * Quantidade2);
    printf("VALOR A PAGAR: R$ %.2lf\n", TOTAL);
    
    return 0;
}

File: Java/1010 - Cálculo Simples.java
/* 
   ==UserScript==
 @name         Cálculo Simples
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1010 - Cálculo Simples
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

import java.util.Scanner;
import java.io.IOException;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scan = new Scanner(System.in);
        int codigoPeca1 = scan.nextInt();
        int numeroPeca1 = scan.nextInt();
        double valorPeca1 = scan.nextDouble();
        
        int codigoPeca2 = scan.nextInt();
        int numeroPeca2 = scan.nextInt();
        double valorPeca2 = scan.nextDouble();
        
        double total = numeroPeca1 * valorPeca1 + numeroPeca2 * valorPeca2;
        
        System.out.println(String.format("VALOR A PAGAR: R$ %.2f", total));
 
    }
 
}
